### DISCUSSION TITLE
Bots slow leveling, can i speed it up?

### DISCUSSION THREAD

**Discussion by boldoran** [mod-playerbots/mod-playerbots#371] [Q&A] [answered]
Hey, ive installed the mod and it seems to be working fine, except that the bots are super slow in leveling.
me and a friend are lvl 12 and the bots are still at lvl 3 and a few of them are 4-5 (i let them all start at 1 to simulate a fresh server)

but they cant keep up and mostly just stand around.
any settings i have wrong? or is this how they work?

also what does: "max ai itterations per tick" do? 

thanks!

**Comment by hermensbas** [comment, marked as answer]
Have look at 

Set kill XP rate for bots (default: 1)
Server XP Rate * AiPlayerbot.KillXPRate
```
AiPlayerbot.KillXPRate = 1
```

**Comment by Mr-Deadbeat** [comment]
with changes to the above settings mentioned by @hermensbas you could complement it with this mod https://github.com/azerothcore/mod-weekend-xp i use it on my server so it doubles the xp rate at weekends not sure if the bots work with the same rules as normal players do but if that is the case it should ramp up their leveling speed some more ;)
